scene/gui/item_list.cpp

```diff
@@ -32,6 +32,9 @@
 #include "core/os/os.h"
 #include "core/project_settings.h"
 
+#define CAN_SELECT(i) ((items[i].selectable) && (!items[i].disabled))
+#define IS_SAME_ROW(i, row) (i / current_columns == row)
```

Comment:
Please `#undef` them after the method.
I'd suggest also moving the `#define` directly in the relevant code branch of `_gui_input` where they'll be used (and undefine at the end of that branch).